### ISSUE TITLE
Communication between Chrome and server

### ISSUE THREAD

**Issue by Geshode** [aiortc/aioquic#124]
Hello @jlaine ,

I have looked through and tried out the examples in the examples folder. 
Now I was wondering, if there are any examples of how to use aioquic for the communications between Chrome and a server?
What I want to try out in the end is file upload and download, but an example for a general request/response communication using aioquic would already be a big help.

Best regards.

**Comment by jlaine** [comment]
Hi!

I'm not sure I understand the question, but you can fire up the example HTTP/3 server and perform requests with Chrome against the server. If you're unsure how to do this, feel free to point Chrome at the publicly accessible demo server:

https://quic.aiortc.org/

**Comment by Geshode** [comment]
@jlaine 

Sorry for replying so late. I had some stuff going on.

I accessed the public demo server and it shows, that I loaded the page with HTTP/3. h3-29 to be exact.

I also ran the example HTTP/3 server locally and when I run the example HTTP/3 client, there is an interaction between the two. Using the ALPN protocol h3-29.

But I am not able to get a connection between Chrome and the local example HTTP/3 server. Even though I access the same URL as I did for the example HTTP/3 client (https://localhost:4433/) and use h3-29.
